components/Navbar.jsx

```
@@ -99,35 +81,20 @@ export default function Navbar() {
-              </a>
+              {navItems.map((item, index) => (
+                <a
+                  key={index}
+                  className="w-full rounded-xl border-[1px] border-transparent"
+                  href={item.href}
+                >
+                  {item.label}
+                </a>
+              ))}
             </div>
             <SheetClose asChild></SheetClose>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)